pom.xml

```diff
@@ -1119,6 +1119,11 @@
                   <property name="format" value="(^[A-Z][0-9]?)$|([A-Z][a-zA-Z0-9]*[T]$)" />
                 </module>
                 <module name="NonEmptyAtclauseDescription" />
+                <module name="RegexpSinglelineJava">
+                  <!-- double escape quotes because checkstyle passes these through another xml parser -->
+                  <property name="format" value="&amp;quot; [+] &amp;quot;" />
+                  <property name="message" value="Unnecessary concatenation of string literals" />
+                </module>
```

Comment:
There are several others of this same type in here. This should be relocated to that group of modules. But, don't worry, I'll take care of it before I merge. :smiley_cat: Thanks!